zhaquirks/smartthings/__init__.py

```diff
@@ -0,0 +1,19 @@
+import zigpy.types as t

```

Comment:
drop this file from the PR and import this from zigpy quirks

Reply:
scratch that... move it to Centralite and we have to change it too... https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/b2c942bd2220e7f5367465c49b8460f9edf70763/devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy#L236